src/components/reusable/card/card.scss

```diff
@@ -0,0 +1,34 @@
+@use '../../../common/scss/global.scss';
+@use '../../../common/scss/mixins.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: inline-block;
+  width: 264px;
+}
+.card {
+  &-wrapper {
+    position: relative;
+    padding: 24px;
+    background-color: var(--kd-color-background-ui-default);
+    outline: 1px solid var(--kd-color-border-light);
+    border-radius: 8px;
+    height: 100%;
+  }
+  &-wrapper-clickable {
+    position: relative;
+    display: inline-block;
+    color: unset;
+    width: inherit;
+    text-decoration: none;
+    background-color: var(--kd-color-background-ui-default);
+    outline: 1px solid var(--kd-color-border-ui);
+    border-radius: 8px;
+    box-shadow: 0px 1px 2px 0px rgba(61, 60, 60, 0.25);
```

Comment:
Pull in [Elevation](https://kyndryl-design-system.github.io/shidoka-foundation/?path=/docs/foundation-elevation--docs) here, now that it's available.

Reply:
Can you give me an example how to use that ?